WoWPro_Leveling/Retail/Neutral/TWW_Dorn.lua

```diff
@@ -401,53 +425,67 @@ T Their Moss, Our Gain|QID|78293|M|58.75,30.30|Z|2248;Isle of Dorn|N|To Head Ran
 T Wanted: The Boroughbreaker|QID|83335|M|54.08,20.06|Z|2339;Dornogal|N|To Peacekeeper Alef.|
 C Return to the Coreway|QID|78545|QO|1|M|31.48,59.69|Z|2339;Dornogal|N|Return to Merrix in Dornogal.|
 T Return to the Coreway|QID|78545|M|32.00,59.61|Z|2339;Dornogal|N|To Merrix.|
 A Recompense|QID|78546|PRE|78545|M|32.00,59.61|Z|2339;Dornogal|N|From Merrix.|
 C Recompense|QID|78546|QO|1|M|31.93,59.35|Z|2339;Dornogal|N|Speak to Adelgonn.|CHAT|
 C Recompense|QID|78546|QO|2|M|31.93,59.35|Z|2339;Dornogal|N|Listen to the debriefing.|CHAT|
-C Recompense|QID|78546|QO|3|M|31.91,59.76|Z|2339;Dornogal|N|Witness the Coreway's restoration.|CHAT|
+C Recompense|QID|78546|QO|3|M|31.91,59.76|Z|2339;Dornogal|N|Tell Merrix "Let's do it."|CHAT|
 T Recompense|QID|78546|M|31.91,59.76|Z|2339;Dornogal|N|To Merrix.|
+A Into the Deeps|QID|80434|M|31.99,59.88|Z|2339;Dornogal|N|From Moira Thaurissan.|
+
+A How Scandalous|QID|80207|M|62.71,21.72|Z|2339;Dornogal|N|From Aldra.|RANK|2|;need to find pre. or if this was available earlier and just missed? - I know it was availablewhen flying into Dornogal before turning in Return to the Coreway-- prolly once per account. (lead 80538?)
+T How Scandalous|QID|80207|M|49.88,49.27|Z|2339;Dornogal|N|To Peacekeeper Kragad.|
+A Seizing Evidence|QID|79521|PRE|80207|M|49.88,49.27|Z|2339;Dornogal|N|From Peacekeeper Kragad.|
+C Seizing Evidence|QID|79521|M|51.43,51.25|Z|2339;Dornogal|N|Pick up the crates on the nearby trolly cart.|H|
+T Seizing Evidence|QID|79521|M|49.88,49.27|Z|2339;Dornogal|N|To Peacekeeper Kragad.|
+A Withheld Information|QID|79522|PRE|79521|M|49.88,49.27|Z|2339;Dornogal|N|To Peacekeeper Kragad.|
+C Withheld Information|QID|79522|M|48.97,49.56;49.81,48.27|CS|Z|2339;Dornogal|N|Question the line workers. Three are in the building behind Kragad, and two outside.|CHAT|
+T Withheld Information|QID|79522|M|49.88,49.27|Z|2339;Dornogal|N|To Peacekeeper Kragad.|
+A Something Hidden|QID|79523|PRE|79522|M|49.88,49.27|Z|2339;Dornogal|N|From Peacekeeper Kragad.|
+C Something Hidden|QID|79523|M|51.43,51.25|Z|2339;Dornogal|N|Back to the crates on the trolly cart, smash them open and take the contents.|
+T Something Hidden|QID|79523|M|49.88,49.27|Z|2339;Dornogal|N|To Peacekeeper Kragad.|
+A Report to the Councilward|QID|79525|PRE|79523|M|49.88,49.27|Z|2339;Dornogal|N|From Peacekeeper Kragad.|
+A Gear Enchanting|QID|83275|M|51.84,48.74|Z|2339;Dornogal|N|From Ellerdin.|;may be profession specific? (as in this toon has enchanting)
+C Gear Enchanting|QID|83275|M|51.84,48.74|Z|2339;Dornogal|N|Use the provided enchantment (+130 haste) on your own ring or the one she gave you.|U|225652|
+T Gear Enchanting|QID|83275|M|51.84,48.74|Z|2339;Dornogal|N|To Ellerdin.|
+C Report to the Councilward|QID|79525|M|55.67,6.01|Z|2339;Dornogal|N|Fly up to the Coucilward's office and approach Merrix's desk.|NC|
+T Report to the Councilward|QID|79525|M|55.67,6.01|Z|2339;Dornogal|N|To Open Ledger.| 
```

Comment:
trailing whitespace

Reply:
Yup: 
`Checking WoWPro_Leveling/Retail/Neutral/TWW_Dorn.lua 2 warnings

    WoWPro_Leveling/Retail/Neutral/TWW_Dorn.lua:16:144: trailing whitespace in a string
    WoWPro_Leveling/Retail/Neutral/TWW_Dorn.lua:483:86: trailing whitespace in a string
`